charts/rancher-logging/0.0.1/charts/fluentd/templates/clusterrole.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.create -}}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "fluentd.fullname" . }}
+  labels:
+    app: {{ include "fluentd.name" . }}
+    chart: {{ template "fluentd.version" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
```

Comment:
`app.kubernetes.io/managed-by` label is the same as `heritage`, it is the recommended labels by [SIG Apps](https://github.com/helm/helm/issues/4335), but it has been released by [helm v2.11.0](https://github.com/helm/helm/releases/tag/v2.11.0).

Reply:
deleted managed-by